deb/Makefile

```diff
@@ -1,14 +1,14 @@
 include ../common.mk
 
-CLI_DIR:=$(realpath $(CURDIR)/../../cli)
-ENGINE_DIR:=$(realpath $(CURDIR)/../../engine)
-PLUGINS_DIR:=$(realpath $(CURDIR)/../plugins)
+CLI_DIR=$(realpath $(CURDIR)/../../cli)
+ENGINE_DIR=$(realpath $(CURDIR)/../../engine)
+PLUGINS_DIR=$(realpath $(CURDIR)/../plugins)
 GITCOMMIT?=$(shell cd $(CLI_DIR) && git rev-parse --short HEAD)
-STATIC_VERSION:=$(shell ../static/gen-static-ver $(ENGINE_DIR) $(VERSION))
+STATIC_VERSION=$(shell ../static/gen-static-ver $(ENGINE_DIR) $(VERSION))
```

Comment:
`STATIC_VERSION` is not used IIRC 🤔

Reply:
Good one; let me check. Perhaps it was a leftover of the image-based builds 🤔